class InternalError

	def main
		.unfinishedMethod('hello!')
		
	def unfinishedMethod(msg  # .error. Unexpected end of file
